examples/OrmliteExample/settings.gradle

```diff
@@ -0,0 +1 @@
+include ':app'
```

Comment:
I would not just create one root module, to keep the project small, just like our gradle and RoboGuice examples.